src/__init__.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+# from https://yann.lecun.com/exdb/mnist/
+def parse_mnist_data(
+    idx_file_training_samples: str,
+    idx_file_training_labels: str,
+    idx_file_test_samples: str,
+    idx_file_test_labels: str,
+) -> tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
+
+    training_samples = parse_mnist_images(idx_file_training_samples)
+    training_labels = parse_mnist_labels(idx_file_training_labels)
+
+    test_samples = parse_mnist_images(idx_file_test_samples)
+    test_labels = parse_mnist_labels(idx_file_test_labels)
+
+    return tuple(training_samples, training_labels, test_samples, test_labels)
```

Comment:
Die Syntax für ein `tuple`-Literal ist `(a, b, c)`, nicht `tuple(a, b, c)`. In Zuweisungen oder Returnstatements kannst du auch die Klammern weglassen.